File: client/src/app/domain/project-status/feedback-provider-status.ts

import { EvalueeDetail } from './evaluee-detail';

export class FeedbackProviderStatus {

    id: number;
    idUser: number;
    username: string;
    mail: string;
    avatar: string;
    status: string;
    pendingFeedbacks: number;
    completedFeedbacks: number;
    evaluees: EvalueeDetail[];
    reminder: boolean;
}

File: projects/src/main/java/com/gire/eval360/projects/domain/dto/FeedbackProviderStatus.java
package com.gire.eval360.projects.domain.dto;

import java.util.List;

import com.gire.eval360.projects.domain.Status;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = false)
public class FeedbackProviderStatus {

	private Long id;
	private Long idUser;
	private String username;
	private String avatar;
	private Status status;
	private Integer pendingFeedbacks;
	private Integer completedFeedbacks;
	private List<EvalueeDetail> evaluees;
	
}
